.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,25 +1,39 @@
 <!--
-Thanks for sending a pull request!
+Thank you for contributing! Please remember to:
 
-Please make sure you've read our contributing guidelines and how to submit a pull request (https://github.com/containers/podman/blob/main/CONTRIBUTING.md#submitting-pull-requests).
+- Read our contributing guidelines: https://github.com/containers/podman/blob/main/CONTRIBUTING.md#submitting-pull-requests
+- Sign your commits with your real name. You can amend previous commits with `git commit -s --amend`.
+-->
+
+#### Description
+<!-- Provide a clear summary of your changes and why they are needed. -->
 
-Finally, be sure to sign commits with your real name. Since by opening
-a PR you already have commits, you can add signatures if needed with
-something like `git commit -s --amend`.
+Fixes:
```

Comment:
I rather not have that by default printed. Many PRs don't fix issues at people will not remove this resulting in me thinking they might made a mistake and forgot to copy the issue number there.